src/operator/controllers/istiopolicy/policy_manager.go

```diff
@@ -402,7 +402,7 @@ func (c *PolicyManagerImpl) createOrUpdatePolicies(
 
 		err = c.client.Create(ctx, newPolicy)
 		if err != nil {
-			if k8serrors.IsConflict(err) {
+			if k8serrors.IsConflict(err) || k8serrors.IsAlreadyExists(err) {
```

Comment:
Shouldn't there be an `updateExisting` logic that we should fallback to ? With `Requeue: true` maybe ?
Asking based on similar logic in netpols creation, might be wrong :)